jetty-server/src/main/java/org/eclipse/jetty/server/HttpConfiguration.java

```diff
@@ -64,6 +64,7 @@
     private boolean _delayDispatchUntilContent = true;
     private boolean _persistentConnectionsEnabled = true;
     private int _maxErrorDispatches = 10;
+    private boolean _useDirectByteBuffers = false;
```

Comment:
@gregw Really great feedback! I agree a single flag is too coarse. I'll also look into the ChannelEndPoint issue as well. I'll mull over some ideas and provide an update once I have something more concrete.